pycbc/pool.py

```diff
@@ -120,6 +120,8 @@ def broadcast(self, fcn, args):
 
     def map(self, f, items):
         return [f(a) for a in items]
+        
+    imap = map
```

Comment:
Ok, added. It's so that we can use imap for our generic pool objects. That allows writing scripts that can be parallelized with imap and also not fail for the single core case.